Pear/Onboarding/DemographicsViewController.swift

```diff
@@ -69,30 +67,37 @@ class DemographicsViewController: UIViewController {
         // Renders the valid graduation years based on current year.
         let currentYear = Calendar.current.component(.year, from: Date())
         let gradYear = currentYear + 4 // Allow only current undergrads and fifth years
-        classSearchFields = (currentYear...gradYear).map { "\($0)" }
-
         classDropdownView = OnboardingSelectDropdownView(delegate: self,
                                                          placeholder: "Class of...",
-                                                         tableData: classSearchFields,
+                                                         tableData: (currentYear...gradYear).map { "\($0)" },
                                                          textTemplate: "Class of")
         classDropdownView.tag = 0 // Set tag to keep track of field selection status.
         view.addSubview(classDropdownView)
 
-        majorDropdownView = OnboardingSearchDropdownView(delegate: self,
-                                                         placeholder: "Major",
-                                                         tableData: majorSearchFields)
+        majorDropdownView = OnboardingSearchDropdownView(
+            delegate: self,
+            placeholder: "Major",
+            tableData: majorSearchFields,
+            searchType: .local
+        )
         majorDropdownView.tag = 1 // Set tag to keep track of field selection status.
         view.addSubview(majorDropdownView)
 
-        hometownDropdownView = OnboardingSearchDropdownView(delegate: self,
-                                                            placeholder: "State",
-                                                            tableData: hometownSearchFields)
+        hometownDropdownView = OnboardingSearchDropdownView(
+            delegate: self,
+            placeholder: "State",
```

Comment:
in EditDemographics, you have the placeholder as "City, State, Country". maybe make them the same?